quantities/measures/Serializer.cs

```diff
@@ -0,0 +1,31 @@
+using Quantities.Prefixes;
+using Quantities.Units;
+
+namespace Quantities.Measures;
+
+internal sealed class Serializer<TUnit>
```

Comment:
I see some advantages to having everything lowercase:
 - it will work well on both case sensitive & case insensitive systems
 - the serialization of a TUnit is concise, so uppercase letters would bring no benefit
 - it's simpler.